packages/opentelemetry-cloud-trace-exporter/package.json

```diff
@@ -38,17 +38,18 @@
   },
   "devDependencies": {
     "@types/mocha": "7.0.2",
-    "@types/nock": "11.1.0",
     "@types/node": "12.12.51",
     "@types/sinon": "9.0.4",
+    "nock": "^12.0.3",
     "codecov": "3.7.2",
     "gts": "2.0.2",
     "mocha": "7.2.0",
     "nyc": "15.1.0",
     "rimraf": "3.0.2",
     "sinon": "9.0.2",
     "ts-mocha": "7.0.0",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "gcp-metadata": "^4.1.0"
```

Comment:
nit: usually we try to list dependencies in the alphabetical order. I would move `gcp-metadata` just above `gts` and `nock` just after `mocha`.

Reply:
Makes sense, thanks for the catch.